### PR TITLE
feat(UI-1063): add organization settings page

### PR DESCRIPTION
**Pull request by J1za**
## Description
Create Organization Settings page
## Linear Ticket
https://linear.app/autokitteh/issue/UI-1063/create-organization-settings-page
## What type of PR is this? (check all applicable)

-   [ ] 💡 (feat) - A new feature (non-breaking change which adds functionality)
-   [ ] 🔄 (refactor) - Code Refactoring - A code change that neither fixes a bug nor adds a feature
-   [ ] 🐞 (fix) - Bug Fix (non-breaking change which fixes an issue)
-   [ ] 🏎 (perf) - Optimization
-   [ ] 📄 (docs) - Documentation - Documentation only changes
-   [ ] 📄 (test) - Tests - Adding missing tests or correcting existing tests
-   [ ] 🎨 (style) - Styles - Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-   [ ] ⚙️ (ci) - Continuous Integrations - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
-   [ ] ☑️ (chore) - Chores - Other changes that don't modify src or test files
-   [ ] ↩️ (revert) - Reverts - Reverts a previous commit(s).

### CODE REVIEW

File: src/components/organisms/settings/organization/organization.tsx [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+import { ModalName } from "@src/enums/components";
+import { SelectOption } from "@src/interfaces/components";
+import { useModalStore } from "@src/store";
+import { organizationSchema } from "@validations";
+
+import { Button, ErrorMessage, Input, Typography } from "@components/atoms";
+import { Select } from "@components/molecules";
+import { DeleteOrganizationModal } from "@components/organisms/settings/organization";
+
+export const Organization = () => {
```
Comment (RonenMars) [inline src/components/organisms/settings/organization/organization.tsx:16]:
```suggestion
export const OrganizationSettings = () => {
```

File: src/components/organisms/settings/organization/organization.tsx [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+		register,
+	} = useForm({
+		resolver: zodResolver(organizationSchema),
+		mode: "onSubmit",
+	});
+	const { openModal } = useModalStore();
+
+	const onSubmit = async () => {};
+
+	const watchedFrequency = useWatch({ control, name: "connection" });
```
Comment (RonenMars) [inline src/components/organisms/settings/organization/organization.tsx:38]:
```suggestion
	const watchedFrequency = useWatch({ control, name: "frequency" });
```

File: src/validations/organization.schema.ts [hunk 3]
```diff
@@ -1,6 +1,18 @@
 import { z } from "zod";
 
+const selectSchema = z.object({
```
Comment (RonenMars) [inline src/validations/organization.schema.ts:3]: We also have it in `src/validations/manualRun.schema.ts`.
Put it in a `shared` folder and import from there.

**Comment by RonenMars** [comment]
Please make the limits boxes 50% width:

image

**Comment by RonenMars** [comment]
Please make bigger margins below and under the "Error notification" title:
image

File: src/validations/organization.schema.ts (hunk 3 above)
Comment (J1za) [reply src/validations/organization.schema.ts:3]: done

File: src/components/organisms/settings/organization/organization.tsx (hunk 2 above)
Comment (J1za) [reply src/components/organisms/settings/organization/organization.tsx:38]: changed

File: src/components/organisms/settings/organization/organization.tsx (hunk 1 above)
Comment (J1za) [reply src/components/organisms/settings/organization/organization.tsx:16]: changed

File: src/components/organisms/settings/organization/organization.tsx [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+				Organization {t("form.settings")}
+			</Typography>
+			<div className="flex flex-wrap gap-10">
+				<div className="w-1/2">
+					<Typography className="mb-2 font-bold" element="h2" size="medium">
+						{t("limits")}
+					</Typography>
+					<div className="flex gap-3">
+						<div className="w-1/2 rounded bg-gray-950 p-3 text-base shadow-xl">
+							<Typography className="mb-1" element="h5">
```
Comment (RonenMars) [inline src/components/organisms/settings/organization/organization.tsx:52]: The title should be in a higher order than the content.
For example, if the title is `h5`, then the content should be `h6` or better `p`.
```suggestion
							<Typography className="mb-1" element="h3">
```

**Comment by RonenMars** [comment]
The form buttons are too close to the form:
image

File: src/components/organisms/settings/organization/organization.tsx (hunk 4 above)
Comment (J1za) [reply src/components/organisms/settings/organization/organization.tsx:52]: changed

**Comment by J1za** [comment]
> The form buttons are too close to the form: image

added more space

File: src/app.tsx [hunk 5]
```diff
@@ -177,7 +177,9 @@ export const App = () => {
 			</Route>
 
 			<Route element={<SettingsLayout />} path="organization-settings">
+				<Route element={<OrganizationSettings />} index />
 				<Route element={<NewOrganization />} path="new" />
+				<Route element={<NewOrganization />} path="new-organization" />
```
Comment (RonenMars) [inline src/app.tsx:182]: Everything is super good, the only comment that I have is that we have 2 routes now for NewOrganization:
<Route element={<NewOrganization />} path="new" />
<Route element={<NewOrganization />} path="new-organization" />

**Review by RonenMars — COMMENTED** [review]
Alt Text

**Comment by RonenMars** [comment]
+ please rebase

File: src/app.tsx (hunk 5 above)
Comment (J1za) [reply src/app.tsx:182]: removed last

**Pull request merged**